pkgs/development/libraries/rdkafka/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkg-config python3 ];
 
-  buildInputs = [ zlib openssl ];
+  buildInputs = [ zlib zstd openssl ];
 
   NIX_CFLAGS_COMPILE = "-Wno-error=strict-overflow";
 
```

Comment:
Can we make patchShebangs point to the files that need to be patched.

Reply:
What files do you think need to be patched? I don't understand the question.

Reply:
I have no clue but if you don't know thats okay, too.

Reply:
To elaborate, I am not aware of any files that need patching and I was surprised that you seemed to think some did without explaining your reasoning. :stuck_out_tongue_closed_eyes:

Reply:
Files are getting patched two lines down but GitHubs excellent review system does not allow me to tag that line.

Reply:
Well, it's not part of this diff, so I certainly appreciate that it doesn't invite foisting random chores on me :grin: Thank you for elaborating though.